### ISSUE TITLE
Playing 4k 360 degree video

### ISSUE THREAD

**Issue by viknesh2020** [oscarmarinmiro/aframe-stereo-component#32]
I have made all the setups as per the instructions. I wanted to play a 4k 360 degree video, which was stereoscopicaly rendered. I tried both left and right eye rendering and top/bottom rendering. None of them rendered the video. It was completely black. However, 1080p videos are playing without any issue. 

Is there anything that I need to do with `<a-entity geometry="primitive: sphere; radius: 100; segmentsWidth: 64; segmentsHeight: 64;"
        material="shader: flat; src: #video;"
        scale="-1 1 1" stereo="eye:left; split: vertical">
    </a-entity>` this? changing these values will adapt a 4k video? I am using Google Chrome to view my video which is locally hosted. I tried in different phones but nothing worked.

**Comment by petesamrogers** [comment]
Yep, I'm having the same issue. Largest video dimensions that is supports seems to be 1920x1920. Anything bigger than that does not work (and that's not big enough to use for a 360 videosphere)

**Comment by jacob-willden** [comment]
I wanted to let everyone know that this issue appears to be resolved now. I tested an upscaled (4096  ×  2048 pixels) version of the sample video (MaryOculus.mp4) with the latest version of aframe-stereo-component.min.js (committed on March 1, 2022) and the following index.html file:
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <title>A-Frame 360 Stereoscopic Video Test</title>
        <script src="https://aframe.io/releases/1.3.0/aframe.min.js"></script>
        <script src="aframe-stereo-component.min.js"></script>
    </head>
    <body>
        <a-scene>
            <video id="myVideo" src="MaryOculus4K.mp4" autoplay loop></video>
            <!-- Camera -->
            <a-camera id="camera" lookcontrols position="0 0 0" stereocam="eye:left;"></a-camera>
            <!-- Video spheres -->
            <a-videosphere src="#myVideo" stereo="eye:left"></a-videosphere>
            <a-videosphere src="#myVideo" stereo="eye:right"></a-videosphere>
        </a-scene>
    </body>
</html>
```
I viewed the video through Chrome (hosted locally), and it worked flawlessly. Hopefully this helps!
